suite-native/module-accounts-management/src/components/AccountSettingsShowXpubButton.tsx

```diff
@@ -13,6 +14,8 @@ export const AccountSettingsShowXpubButton = ({ accountKey }: { accountKey: stri
         selectAccountByKey(state, accountKey),
     );
 
+    const isAuthenticityCheckFailed = useSelector(selectHasFirmwareAuthenticityCheckHardFailed);
```

Comment:
nitpick: I know it's a long name, but descriptive. And I like the convention of naming the result of selector by the selector itself. It's like that in all the other places where selectHasFirmwareAuthenticityCheckHardFailed is used.
```suggestion
    const hasFirmwareAuthenticityCheckHardFailed = useSelector(selectHasFirmwareAuthenticityCheckHardFailed);
```

Reply:
good nitpick! I've updated it.